package.json

```diff
@@ -225,6 +225,7 @@
       "atom-beautify:beautify-language-go",
       "atom-beautify:beautify-language-golang template",
       "atom-beautify:beautify-language-fortran",
+      "atom-beautify:beautify-language-fuse",
```

Comment:
Is Fuse the Atom-Beautify language? I thought it was `UX Markup`.

Note that these should be automatically generated/added, so if you find yourself adding this option, it should actually not be there 😜 .